node/src/actors/chain_manager/mining.rs

```diff
@@ -1,4 +1,11 @@
-use std::{cmp::Ordering, convert::TryFrom};
+use std::{
+    cmp::Ordering,
+    convert::TryFrom,
+    sync::{
+        atomic::{AtomicU16, Ordering::Relaxed},
```

Comment:
Better to use `Ordering::Relaxed` in the code

```suggestion
        atomic::{AtomicU16, Ordering},
```